examples/jsm/webxr/ARButton.js

```diff
@@ -107,7 +107,12 @@ class ARButton {
 					if ( navigator.xr.offerSession !== undefined ) {
 
 						navigator.xr.offerSession( 'immersive-ar', sessionInit )
-							.then( onSessionStarted );
+							.then( onSessionStarted )
+							.catch( ( err ) => {
+
+								console.log( err );
```

Comment:
> This PR will catch the failed promises and show them as console warnings

I've seen now that `log()` is used instead of `warn()`. So the code not really prints a warning, just a straight log.

Is that actually intended?

Reply:
I didn't want it to appear as an error so decided to use `log()`. Would `warn()` be better?

Reply:
I would say yes. According to the design there should only be one XR button present on a page. If that is not true, we should warn about this situation.